.env.example

```
@@ -0,0 +1,23 @@
+NODE_ENV=development # or 'production'
+PORT=3000
+LISTEN_ADDRESS=0.0.0.0
+
+SESSION_SECRET=not-secure-change-me # an UUID or `openssl rand -hex 32` or something similar
+SESSION_MAX_AGE= 604800000 # TTL in ms. example is equal to 7 weeks (1000 * 60 * 60 * 24 * 7)
+DISABLE_SECURE_COOKIE=true
+
+# File Upload Configuration
+UPLOAD_DIR=/home/user/code/web/cbt-workbook/uploads
+UPLOAD_PATH=/uploads
+EXTERNAL_UPLOAD_URL=https://cdn.example.com # Uncomment in production to use external CDN or NGINX for file serving
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The EXTERNAL_UPLOAD_URL key should go before the UPLOAD_DIR key

(UnorderedKey)